UI/scripts/view.js

```diff
@@ -0,0 +1,90 @@
+
+const dateConfig = {
+  year: 'numeric',
+  month: 'numeric',
+  day: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+};
+
+class View {
+  constructor(model, isLoggedUser) {
+    this.photoPosts = model;
+    this.isLoggedUser = isLoggedUser;
+  }
+
+  addPost(post) {
+    if (this.photoPosts.addPhotoPost(post)) {
+      const wrapper = document.querySelector('#wrapper');
+      const newPost = View.getNodeWithHtml(post);
+      wrapper.insertAdjacentElement('afterbegin', newPost);
+      return true;
+    }
+    return false;
+  }
+
+  deletePost(id) {
+    if (this.photoPosts.removePhotoPost(id)) {
+      const childNode = document.querySelector(`[data-id="${id}"]`);
+      if (childNode) {
+        childNode.parentNode.removeChild(childNode);
+      }
+      return true;
+    }
+    return false;
+  }
+
+  editPost(id, edits) {
+    if (this.photoPosts.editPhotoPost(id, edits)) {
+      const childNode = document.querySelector(`[data-id="${id}"]`);
+      if (childNode) {
+        const editedChildNode = View.getNodeWithHtml(this.photoPosts.getPhotoPost(id));
+        childNode.parentNode.replaceChild(editedChildNode, childNode);
+      }
+      return true;
+    }
+    return false;
+  }
+
+  showPosts() {
+    this.photoPosts.getPhotoPosts().reverse().forEach((post) => {
+      const wrapper = document.querySelector('#wrapper');
+      const newPost = View.getNodeWithHtml(post);
+      wrapper.insertAdjacentElement('afterbegin', newPost);
+    });
+  }
+
+  displayHeaderElements() {
+    if (this.isLoggedUser) {
+      document.querySelector('#sign-in').style.display = 'none';
+      document.querySelector('.user-name').style.display = 'inline-block';
+      document.querySelector('#log-out').style.display = 'inline-block';
+    }
+  }
+
+  static getNodeWithHtml(post) {
+    const newPost = document.createElement('div');
+    newPost.innerHTML = `<div class = "post">
```

Comment:
Cosmetic: +1 if it will be redone using HTMLTemplate approach.

Reply:
```suggestion
<template id="post-template">
    <div class = "post">
         <p class = "postInfo">
              <b data-target="post.author"></b>
         </p>
         ...
```
How I can set post.author value into `<b>` ,trying to use ur example(http://alan-js-learn.herokuapp.com/pen/dom/template-example-2) : 
```suggestion
const placeholders = postNode.querySelectorAll('[data-target]');
    [].forEach.call(placeholders || [], (phElement) => {
      const key = phElement.getAttribute('data-target');
      phElement.textContent = `${key}`; //mistake here
    });
```